tests/validations/test_unique_valid_name.py

```diff
@@ -70,6 +70,27 @@ def test_duplicate_semantic_model_name(  # noqa: D
         ).checked_validations(model)
 
 
+def test_metric_name_validity(  # noqa: D
+    simple_semantic_manifest__with_primary_transforms: PydanticSemanticManifest,
+):
+    validator = SemanticManifestValidator[PydanticSemanticManifest](
+        [UniqueAndValidNameRule[PydanticSemanticManifest]()]
+    )
+
+    # Shouldn't raise an exception
+    validator.checked_validations(simple_semantic_manifest__with_primary_transforms)
+
+    # Should raise an exception
+    compied_manifest = deepcopy(simple_semantic_manifest__with_primary_transforms)
+    metric = compied_manifest.metrics[0]
```

Comment:
compied copied compied copied. 😛

Reply:
Corrected in: [b99cc42](https://github.com/dbt-labs/dbt-semantic-interfaces/pull/184/commits/b99cc4272bbf6b5f6b54f4d5562016329d7d6eee)
Will be fixed up into: [f0b1389](https://github.com/dbt-labs/dbt-semantic-interfaces/pull/184/commits/f0b138925841f792c62f83091ef48991f7e40dfd)